documentation/database/sqlserver.md

```diff
@@ -179,6 +180,11 @@ To use the various authentication types, amend your JDBC URL to set the `authent
 - For Active Directory With Password set `authentication=ActiveDirectoryPassword`
   - e.g: <code>jdbc:sqlserver://<i>host</i>:<i>port</i>;databaseName=<i>database</i>;authentication=ActiveDirectoryPassword</code>
   - You must also supply a username and password with Flyway's `user` and `password` configuration options
+- For Active Directory Interactive set `authentication=ActiveDirectoryInteractive`
+  - e.g: <code>jdbc:sqlserver://<i>host</i>:<i>port</i>;databaseName=<i>database</i>;authentication=ActiveDirectoryInteractive</code>
+  - This will being an interactive process which expects user input (e.g. a dialogue box), so it's not recommended automated environments
```

Comment:
`being` sounds like it should be `begin`? It also sounds like a word is missing between `recommended` and `automated`, like `in`?

Reply:
:o Done, thanks